app/models/manageiq/providers/ibm_cloud/power_virtual_servers/cloud_manager/event_target_parser.rb

```diff
@@ -0,0 +1,34 @@
+class ManageIQ::Providers::IbmCloud::PowerVirtualServers::CloudManager::EventTargetParser
+  attr_reader :ems_event
+
+  def initialize(ems_event)
+    @ems_event = ems_event
+  end
+
+  def parse
+    target_collection = InventoryRefresh::TargetCollection.new(
+      :manager => ems_event.ext_management_system,
+      :event   => ems_event
+    )
+
+    case ems_event[:event_type]
+    when /^pvm-instance\.create/
+      target_collection.add_target(
+        :association => :ext_management_system,
```

Comment:
@agrare I'm still struggling to target the entire ems (haven't been able to find a similar situation elsewhere in MIQ).

Reply:
Yeah I don't believe any other `EventTargetParser`s do this let me play around a bit and let you know how you'd go about doing this

Reply:
Thanks @agrare - implemented here: 572010c